src/app/workflow/dag/cwl-viewer/cwl-viewer.component.ts

```diff
@@ -27,13 +31,12 @@ import { Subject } from 'rxjs';
   styleUrls: ['./cwl-viewer.scss']
 })
 
-export class CwlViewerComponent implements OnInit, OnDestroy {
+export class CwlViewerComponent implements OnInit, OnChanges, OnDestroy {
 
   @Input() workflow: ExtendedWorkflow;
```

Comment:
check whether this is still used (since you have the actual extendedWorkflow now).  if it's not used, remove it.  if it is, change the type to Workflow.

Reply:
Removed the variable and looks like all my imports are being used.

Reply:
Check extendedWorkflowService and extendedWorkflowStore